### PR TITLE
You can now change your underwear at mirrors + prompt

### PR DESCRIPTION
**Pull request by SonixApache**
🆑 
 - rscadd: You can now change your underwear at mirrors.
 - tweak: You now get a prompt which thing to change at mirrors.

### CODE REVIEW

**Comment by ghost** [comment]
where does the new underwear come from tho

**Comment by bathosbathtime** [comment]
Clearly the mirror has a medicine cabinet behind it.  Full of underwear.

**Comment by SonixApache** [comment]
same place as where the floor-length braids and full beards come from, nanomachines

feel free to sprite a dresser and map it into every map, i'll port the functionality

**Comment by utilitas-** [comment]
There used to be a dresser structure, at least on some server, which exists solely to change underwear. That might be a handier solution than having to go through 4 dialog boxes at a mirror.

**Comment by strawberrytan** [comment]
dressers are vampire powercrepe

**Comment by ShiftyRail** [comment]
Kinky

**Comment by SirSkeleto** [comment]
qol suggestion: instead of doing beard, hair, underwear, just have it let you choose which one instead of having to pick all 3 each time

**Comment by Lokiamis** [comment]
Underwear slot when

**Comment by SonixApache** [comment]
image
ok

File: code/game/objects/structures/mirror.dm [hunk 1]
```diff
@@ -37,29 +37,51 @@
...
-		var/list/species_facial_hair = valid_sprite_accessories(facial_hair_styles_list, H.gender, (H.species.name || null))
-		if(species_facial_hair.len)
-			var/new_style = input(user, "Select a facial hair style", "Grooming")  as null|anything in species_facial_hair
-			if(userloc != H.loc)
-				return	//no tele-grooming
-			if(new_style)
-				H.my_appearance.f_style = new_style
-				H.update_hair()
+		//handle facial hair
+		if(which == "Beard")
```
Comment (Kurfursten) [inline code/game/objects/structures/mirror.dm]: just do a switch

File: code/game/objects/structures/mirror.dm [hunk 2]
```diff
@@ -37,29 +37,51 @@
 					return
 		var/userloc = H.loc
 
+		
+		var/which = alert("What would you like to change?", "Appearance", "Hair", "Beard", "Undies")
+
+		if((!which) || (userloc != H.loc))
```
Comment (Kurfursten) [inline code/game/objects/structures/mirror.dm]: an adjacency check would be preferred I think, as long as you remain in slapping range of that mirror

File: code/game/objects/structures/mirror.dm (hunk 1 above)
Comment (SonixApache) [reply code/game/objects/structures/mirror.dm]: wilco

File: code/game/objects/structures/mirror.dm (hunk 2 above)
Comment (SonixApache) [reply code/game/objects/structures/mirror.dm]: wilco

**Pull request merged**